sqs/src/main/scala/akka/stream/alpakka/sqs/scaladsl/SqsSource.scala

```diff
@@ -25,7 +28,10 @@ object SqsSource {
   def apply(
       queueUrl: String,
       settings: SqsSourceSettings = SqsSourceSettings.Defaults
-  )(implicit sqsClient: SqsAsyncClient): Source[Message, NotUsed] =
+  )(implicit sqsClient: SqsAsyncClient): Source[Message, NotUsed] = {
+    val controlledThrottling =
+      new ControlledThrottling[ReceiveMessageRequest](Clock.systemDefaultZone(), settings.waitTimeSeconds.seconds)
```

Comment:
Would there be a reason to introduce another setting, to configure it independently of `waitTimeSeconds`?

Reply:
Fair point. I think that in case of using throttling it might make more sense to have a configuration to switch "autothrottling" off, so that clients that are interested in low latencies could still use the library in the same way it works now.